diff_cover/diff_quality_tool.py

```diff
@@ -142,6 +143,10 @@ def parse_quality_args(argv):
         "--exclude", metavar="EXCLUDE", type=str, nargs="+", help=EXCLUDE_HELP
     )
 
+    parser.add_argument(
+        "--include", metavar="INCLUDE", default=["**/*.py"], nargs="+", type=str, help=INCLUDE_HELP
+    )
```

Comment:
@kasium This default is why the test is failing.

Nothing is matching the glob so the code never checks if the tool is installed. 

I also dont think this default is appropriate. This tools is not just used with python code. I think we should only be checking the glob when provided.

Reply:
Default changed